dev/sdk_python.go

```diff
@@ -28,28 +25,37 @@ type PythonSDK struct {
 	Dagger *DaggerDev // +private
 }
 
+func (t PythonSDK) dev(opts ...dagger.PythonSDKDevOpts) *dagger.PythonSDKDev {
+	src := t.Dagger.Source.Directory(pythonSubdir)
+	return dag.PythonSDKDev(src, opts...)
+}
+
+func (t PythonSDK) directory(dist *dagger.Directory) *dagger.Directory {
```

Comment:
What does this do?

Reply:
Directories returned by `PythonSDKDev` are relative to `sdk/python`, so in the context of the root `/dev` module, this puts those directories under that subdir.